test/test_rosbag/test/test_bag.py

```diff
@@ -312,6 +312,14 @@ def test_get_time(self):
             self.assertEquals(start_stamp, 0.0)
             self.assertEquals(end_stamp, 99.0)
 
+    def test_get_time_empty_bag(self):
+        """Test for issue #657"""
+        fn = '/tmp/test_get_time_emtpy_bag.bag'
+
+        with rosbag.Bag(fn, mode='w') as bag:
+            self.assertRaises(rosbag.ROSBagException, bag.get_start_time)
+            self.assertRaises(rosbag.ROSBagException, bag.get_end_time)
```

Comment:
You can assert that it raises the correct `ROSBagException` as well by checking the error message with: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaisesRegexp